rclcpp/include/rclcpp/node.hpp

```diff
@@ -851,6 +852,7 @@ class Node : public std::enable_shared_from_this<Node>
    * \return The previous callback that was registered, if there was one,
    *   otherwise nullptr.
    */
+  [[deprecate("use add_on_set_parameters_callback(OnParametersSetCallbackType callback) instead")]]
```

Comment:
The correct attribute name is [deprecated](https://en.cppreference.com/w/cpp/language/attributes/deprecated).

```suggestion
  [[deprecated("use add_on_set_parameters_callback(OnParametersSetCallbackType callback) instead")]]
```

I missed it, but the compiler didn't.